mobile/test/java/integration/AndroidEngineExplicitFlowTest.java

```diff
@@ -481,7 +481,7 @@ public void post_multipleRequests_randomBehavior() throws Exception {
           mockWebServer.enqueue(new MockResponse().setBody("hello, world"));
           RequestScenario requestScenario =
               new RequestScenario()
-                  .setHttpMethod(RequestMethod.GET)
+                  .setHttpMethod(RequestMethod.POST)
```

Comment:
why changing to a POST?  and should the url used on the line below be changed too?

Reply:
Because the test name is `post_` and there's already the one that tests `GET` in https://github.com/envoyproxy/envoy/blob/8c635c44998b26939e900a1f0fe27061d14ca40c/mobile/test/java/integration/AndroidEngineExplicitFlowTest.java#L126

I checked all the other tests and they all are correct, except this.

Reply:
Oh you mean the path. Done, I updated the path to `post/`, but I don't think it matters.